src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+class Square extends Figure {
+    private int side;
+
+    public Square(String color, int side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: "
+                + " the side is of length: "
+                + side
+                + " and the color is"
+                + color);
```

Comment:
The 'draw' method's message is not very clear and could be improved for better readability. Additionally, there is a missing space before the color in the message.